spec/envied_spec.rb

```diff
@@ -116,12 +110,13 @@ def envied_require(*args)
       end
     end
 
+    # TODO: review needed, doesn't make sense?
     context 'bug: default value "false" is not coercible' do
```

Comment:
@eval I'm not sure if you remember but figured I'd ask about this test case because the description and what its testing didn't make sense to me.

Reply:
Still though do you know what this is doing exactly? Is it testing "true" simply because "false" can't be tested against?

Reply:
I realize now the test data is using the boolean `true` value as the default rather than the string `"true"` version but in any case of `"true"`, `true`, `"false"` or `false` the test passes.

I'll respond on removing defaults support in #34 instead of here.